manifests/prepare/package.pp

```diff
@@ -17,20 +17,25 @@
   if $puppet_agent::params::master_agent_version != $::aio_agent_version {
     $pe_server_version = pe_build_version()
 
-    $source = "puppet:///pe_packages/${pe_server_version}/${::platform_tag}/${package_file_name}"
+    if $::osfamily == 'windows' {
+      $tag = $::puppet_agent::arch ? {
+        'x64' => 'windows-x86_64',
+      }
+      $source = "puppet:///pe_packages/${pe_server_version}/${tag}/${package_file_name}"
```

Comment:
This is a bug in pe_repo https://tickets.puppetlabs.com/browse/PE-13759 (created by PE-12030)